### ISSUE TITLE
Postman Mac Beta - XML pretty print not working

### ISSUE THREAD

**Issue by P120D1GY** [a85/POSTMan-Chrome-Extension#1697]
XML pretty print is not working with Postman Mac Version 4.0.0-beta.2 (4.0.0-beta.2)

**Comment by abhijitkane** [comment]
@P120D1GY Can you try hitting httpbin.org/xml in the app? The XML Pretty printing seems to work fine for me.

**Comment by stormposse** [comment]
XML Pretty Print does not work for me either in the Mac v4.0.0-beta.2 version. The XML from httpbin.org/xml is already 'pretty-printed' in the raw text of the response and Postman seems to show it exactly as-is, but an XML string like:

```
<?xml version="1.0" encoding="UTF-8"?><A><B/><C date="1988-07-14" name="fox1"/><D/></A>
```

is not pretty printed. It worked OK in v3.

**Comment by P120D1GY** [comment]
Looks like this was fixed in 4.0.0-beta.3.

@stormposse:
Does it work for you as well after today's update?

**Comment by P120D1GY** [comment]
Marking issue as closed, 4.0.0-beta.3 resolve the issue.
